<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'sylius.authentication.success_handler' shared service.

include_once \dirname(__DIR__, 4).'/vendor/symfony/security/Http/Authentication/AuthenticationSuccessHandlerInterface.php';
include_once \dirname(__DIR__, 4).'/vendor/symfony/security/Http/Util/TargetPathTrait.php';
include_once \dirname(__DIR__, 4).'/vendor/symfony/security/Http/Authentication/DefaultAuthenticationSuccessHandler.php';
include_once \dirname(__DIR__, 4).'/vendor/sylius/sylius/src/Sylius/Bundle/UserBundle/Authentication/AuthenticationSuccessHandler.php';

return $this->privates['sylius.authentication.success_handler'] = new \Sylius\Bundle\UserBundle\Authentication\AuthenticationSuccessHandler(($this->privates['security.http_utils'] ?? $this->load('getSecurity_HttpUtilsService.php')), []);
